packages/lesswrong/server/vulcan-lib/apollo-ssr/renderPage.tsx

```diff
@@ -447,5 +450,5 @@ const renderRequest = async ({req, user, startTime, res, clientId, userAgent}: R
 }
 
 const printTimings = (timings: RenderTimings): string => {
```

Comment:
Maybe this should be renamed to something like `formatTimings`? I would 100% expect a `printTimings` function to log something to the console and return void. (Sorry - I know this wasn't added in this PR but I've never noticed it before).